lib/dom.js

```diff
@@ -89,7 +95,8 @@ define(function() {
 
   // Elements that can be jumped to directly from a string
   var selectorShortcut = {
-    document: document.documentElement,
+    document: document,
+    window: window,
```

Comment:
Hmm. I was trying to match what jQuery does but it looks like I got it wrong. So you're right, we can get rid of them and add an `html` key.